src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of
+ * related CSS properties (see an exampl
+ * in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitBySem = sourceString.split(';');
+  const trimed = splitBySem.map(
+    element => element.trim().split(':').map(el => el.trim()));
```

Comment:
```suggestion
  const trimed = splitBySem.map(
    element => element
      .trim()
      .split(':')
      .map(el => el.trim())
  );
```
![image](https://user-images.githubusercontent.com/96409563/224925996-bc55272a-c7b1-470e-855b-1fa8016e6fc5.png)